### ISSUE TITLE
protoc fails with Murmur RPC protocol

### ISSUE THREAD

**Issue by K900** [vmagamedov/grpclib#90]
I'm trying to connect to the Murmur RPC service over gRPC with grpclib, but compiling [the .proto file](https://github.com/mumble-voip/mumble/blob/master/src/murmur/MurmurRPC.proto) fails with

```
Traceback (most recent call last):
  File "/home/k900/grpc_scratch/venv/bin/protoc-gen-python_grpc", line 11, in <module>
    load_entry_point('grpclib==0.3.0', 'console_scripts', 'protoc-gen-python_grpc')()
  File "/home/k900/grpc_scratch/venv/lib/python3.7/site-packages/grpclib/plugin/main.py", line 206, in main
    request_type=types_map[method.input_type],
KeyError: '.MurmurRPC.Server.Query'
```

Using Python 3.7, 
```
grpcio==1.23.0
grpcio-tools==1.23.0
grpclib==0.3.0
```

**Comment by vmagamedov** [comment]
Looks like a bug, you encountered it because MurmurRPC uses nested message types as arguments, this wasn't expected.

**Comment by vmagamedov** [comment]
Fix is released in `grpclib==0.3.1rc2`
